src/test/java/com/iexec/worker/compute/pre/PreComputeServiceTests.java

```diff
@@ -26,18 +26,20 @@
 import com.iexec.common.task.TaskDescription;
 import com.iexec.common.tee.TeeEnclaveConfiguration;
 import com.iexec.common.tee.TeeEnclaveConfigurationValidator;
+import com.iexec.common.tee.TeeEnclaveProvider;
 import com.iexec.sms.api.TeeSessionGenerationError;
 import com.iexec.sms.api.TeeSessionGenerationResponse;
 import com.iexec.worker.compute.ComputeExitCauseService;
-import com.iexec.worker.compute.TeeWorkflowConfiguration;
+import com.iexec.sms.api.TeeWorkflowConfiguration;
```

Comment:
Little issue in imports order here

Reply:
Right, please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/464/commits/2f68229c991043391f6f042d5760d892616a5b8e.